dev/TabView/InteractionTests/TabViewTests.cs

```diff
@@ -804,6 +804,44 @@ void CloseTabAndVerifyWidth(string tabName, int expectedValue, string expectedSc
             }
         }
 
+        [TestMethod]
+        public void VerifyHeaderSizeWhenClosingLastTab()
```

Comment:
>VerifyHeaderSizeWhenClosingLastTab [](start = 20, length = 34)

Does this test fail without your change?

Reply:
Yes, the TabViewItem `Width` gets set to ~79 without the fix (because all tabs are compressed into the visible viewport)